packages/integration-tests/src/e2e-flow/setup_package_manager.ts

```diff
@@ -0,0 +1,120 @@
+import { execa } from 'execa';
+import * as fsp from 'fs/promises';
+import { existsSync } from 'fs';
+import * as path from 'path';
+import * as os from 'os';
+
+type PackageManager = 'npm' | 'yarn-classic' | 'yarn-modern' | 'pnpm';
+type PackageManagerExecutable = 'npx' | 'yarn' | 'pnpm';
+
+const initializeNpm = async () => {
```

Comment:
looks like all of the other functions are pointing the package registry to localhost. does this function need to do the same?

Reply:
Good catch. Our `vent` does it already for `npm`

Reply:
Since we aren't using vend for local testing anymore, maybe it makes sense to rework that command to only start the local registry and publish, and then these methods are responsible for updating package manager config to point to localhost? I'm not sure, I can see arguments both ways

Reply:
I vote for the laizier one 😁